packages/CollabCodeEdit-Client.package/CCEClient.class/instance/startHeartBeat.st

```diff
@@ -0,0 +1,3 @@
+heartbeat
+startHeartBeat
```

Comment:
You're using camelcase inconsistent: Decide for either `heartBeat` or `heartbeat`